eox_core/api_schema.py

```diff
@@ -0,0 +1,58 @@
+"""
+Swagger view generator
+"""
+from django.conf import settings
+from django.conf.urls import url
+from drf_yasg.generators import OpenAPISchemaGenerator
+from drf_yasg.openapi import SwaggerDict
+from drf_yasg.views import get_schema_view
+from edx_api_doc_tools import get_docs_cache_timeout, make_api_info
+from rest_framework import permissions
+
+from eox_core.api.v1 import views
+
+
+class APISchemaGenerator(OpenAPISchemaGenerator):
+    """
+    Schema generator for eox-core.
+
+    Define an exclusive base path to perform requests for each endpoint and a
+    specific security definition using oauth without overwritting project wide
+    settings.
+    """
+
+    def get_schema(self, request=None, public=False):
+        schema = super().get_schema(request, public)
+        schema.base_path = '/eox-core/api/v1/'
```

Comment:
Can you change this with

`eox-core/api/?

so it can include future versions?

Reply:
If I do that, the requests are made against `/eox-core/api/<endpoint>/` instead of `/eox-core/api/v1/<endpoint>/`. I didn't find a way to fix that.

Reply:
Even if I do that it drops the prefix.

Reply:
Ok, then I think that we can keep it at this way for now